syntaxes/samarium.tmLanguage.json

```
@@ -120,7 +120,7 @@
                 },
                 {
                     "name": "keyword.control.samarium",
-                    "match": "(\\*|->|(?<!<)<-|\\.{2,}|@|\\?|!|\\?~>|,,|#|,\\.,|'|\\$|%|<=)"
+                    "match": "(\\*|->|(?<!<)<-|\\.{2,}|@|\\?|!|\\?~>|,,|#|,\\.,|'|\\$|%|<=|~'*)"
```

Keep in mind that `*` is a quantifier, so it needs to be escaped
```suggestion
                    "match": "(\\*|->|(?<!<)<-|\\.{2,}|@|\\?|!|\\?~>|,,|#|,\\.,|'|\\$|%|<=|~'\\*)"
```

---

thanks, I have learned so much from these two PR's .

---

Glad to hear that!